recipes/sortmerna/3.0.0/build.sh

```diff
@@ -0,0 +1,4 @@
+mkdir -pv $PREFIX/bin
+bash build.sh
```

Comment:
It seems there is no build.sh file in the source dir?

Reply:
yeah, building now depends on cmake, i will make the necessary modifications :-)